### PR TITLE
Update Dev Guide > Code Considerations file

### PR DESCRIPTION
**Pull request by sarina**
Addresses #401

Rendered: https://docsopenedxorg--424.org.readthedocs.build/en/424/developers/references/developer_guide/process/code-considerations.html

### CODE REVIEW

File: source/developers/references/developer_guide/process/code-considerations.rst [hunk 1]
```diff
@@ -9,15 +9,91 @@ as they are building new, or modifying existing, functionality.
...
+API documentation, decision records (ADRs and OEPs), and README files. Of
+course, you should always provide detailed, informative comments within your
+code and excellent docstrings for your functions and classes!
+
+If you have developed a new feature, created an XBlock, or otherwise see that a
+topic is missing in this documentation, you can create one or more new pages in
+this document and submit a pull request to have the content added. Follow the
+guidelines in :doc:`../../../documentors/references/doc_guidelines`.
+
+**TODO** How do I add a note about my new feature for the next named release release notes?
```
Comment (sarina) [inline source/developers/references/developer_guide/process/code-considerations.rst:46]: @pdpinch I know you're not writing release notes this time around, but I'm not sure who else to ask -- if I develop a new feature, is there a good way of letting the release note writer(s) know about it?

**Comment by sarina** [comment]
@kdmccormick - can you take a pass at this? I think this is the doc where we can add stuff about proper naming (re: retro discussion) but it would make sense to merge this first, then add that in its own PR.

File: source/developers/references/developer_guide/links.rst [hunk 2]
```diff
@@ -264,6 +264,8 @@
 .. _jscs: https://www.npmjs.org/package/jscs
 .. _Waffle: http://waffle.readthedocs.io/en/latest
 .. _Waffle documentation: http://waffle.readthedocs.io/en/latest
+.. _OEP-17 Feature Toggles: https://open-edx-proposals.readthedocs.io/en/latest/best-practices/oep-0017-bp-feature-toggles.html
```
Comment (feanil) [inline source/developers/references/developer_guide/links.rst:267]:
```suggestion
.. _OEP-17 Feature Toggles: https://docs.openedx.org/projects/openedx-proposals/en/latest/best-practices/oep-0017-bp-feature-toggles.html
```

File: source/developers/references/developer_guide/process/code-considerations.rst (hunk 1 above)
Comment (feanil) [reply source/developers/references/developer_guide/process/code-considerations.rst:46]: Under the [Release Planning](https://openedx.atlassian.net/wiki/spaces/COMM/pages/13205845/Open+edX+Release+Planning) wiki page, there is usually a page for the next release, for example the [Redwood Wiki Page](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3890380898/Next+Release+Redwood).  This is the place to put any changes related to deprecations, operational notes or new features that you want to highlight.  The release notes person usually looks at this as a part of the process for generating notes.

Comment (sarina) [reply source/developers/references/developer_guide/process/code-considerations.rst:46]: Yeah, oddly I already added that later in this page.

**Review by feanil — APPROVED** [review]

**Pull request merged**